### ISSUE TITLE
Deprecated function buttonset()

### ISSUE THREAD

**Issue by fretzl** [Studio-42/elFinder#3680]
With jQuery v3.7.1 and jQuery UI v1.14.0 there is this persistent warning in the browser console each time elFinder initializes:

`TypeError: n.find(...).buttonset is not a function`

According to the docs (https://api.jqueryui.com/buttonset/) the function `buttonset()` has been deprecated since jQuery UI v1.12 and should be replaced by `controlgroup()`.

The warning is indeed fixed by replacing a few occurrences of `.buttonset()` with `.controlgroup()` in `elfinder.full.js` and `elfinder.min.js`.

**Comment by dleffler** [comment]
E.g., 2 occurrences in searchbutton.js and rename.js?

**Comment by fretzl** [comment]
No, in `elfinder.full.js`  (lines 20426 and 31998) and consequently in `elfinder.min.js`
